server/service.go

```diff
@@ -50,7 +51,7 @@ func NewDefaultHTTPServerTimeouts() HTTPServerTimeouts {
 // BoostService TODO
 type BoostService struct {
 	listenAddr string
-	relays     []RelayEntry
+	relays     []*RelayEntry
```

Comment:
Nit: why a pointer instead of a value? `RelayEntry` is a bunch of pointers and a string, it should be just fine as a value, and value semantics are usually preferable.

Reply:
Good point, will update